transcom/mymove
# 19782 shipment cancellation date validation Backend INT

## [B-19782](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A946146)

## Summary

This PR prevents the TOO from cancelling any shipment that's 'actualPickupdate' is on or after the request date.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Make a HHG 
2. Login as SC to edit and submit move 
3. Login as Prime to update shipment with an 'actualPickupDate' on or after today
4. Login as TOO and try to cancel the move
5. Confirm the Update error message

### Backend


## Screenshots
<img width="1145" alt="Screenshot 2024-06-05 at 1 23 44 PM" src="https://github.com/transcom/mymove/assets/146969726/170aa7b2-aa9b-4614-a19e-34d2780cb8f6">
